### DISCUSSION TITLE
Reference Systems and Scales

### DISCUSSION THREAD

**Discussion by neilotte** [CommonCoreOntology/CommonCoreOntologies#784] [Ontology Terms and Class Schema]
I'm wondering where to place [Likert Scale](https://en.wikipedia.org/wiki/Likert_scale) within CCO hierarchy. 

An example Likert scale might include the values: 'Strongly Agree, Agree, Neither Agree nor Disagree, Disagree, Strongly Disagree'. A study participant might then respond 'Agree', affirming one of these values.  

As with measurement units, study participant responses only make sense within the context of a scale (e.g. Were we to use a 7 pt scale, the string 'Agree' would have a different meaning). However, cco:MeasurementUnit is defined 'A Descriptive Information Content Entity that describes a definite magnitude of a physical quantity, defined and adopted by convention and/or by law, that is used as a standard for measurement of the same physical quantity.' For this reason, I'm assuming this excludes Likert Scales, questionnaires, and other survey instruments. I think this makes sense. 

While looking for a good home for Likert scale, I found the following two classes:  
- cco:Reference System: "A Descriptive Information Content Entity that describes a set of standards for organizing and understanding data of the specified type or domain." 
- cco:Priority Scale: "A Reference System that describes a scale used to rank or identify the importance of entities of the specified type." 

Some thoughts: 
1. The definition of priority scale says it _describes a scale_, but presumably a priority scale is a scale ('scale' is undefined). Seems a little circular. 
2. From these definitions, it seems like cco:MeasurementUnit should be a subclass of cco:ReferenceSystem.  
3. If the reference system is a standard, then it shouldn't also 'describe a set of standards' -- rather it should _be_ a standard, and then it seems it should _prescribe_ rather than _describe_ "the organization or understanding of data."
4. It would be nice if there were a similar pattern to the primary one CCO uses for measurements (p. 7 Modeling Information with the CCO 1.0) for scales. That is, in place of the instance of cco:MeasurementUnit and its relation 'has measurement unit' to the Information Bearing Entity, there was instance a parent relation for 'has measurement unit' that was something like 'has reference system', which could be used to relate other reference system (e.g. scales, questionnaires, etc) to an information content entity. 

Does any of this sound correct, or is there another way you would advise accommodating datasets containing responses to psychometric data?

**Comment by mark-jensen** [comment]
@neilotte Interesting question. A single Likert response is an ordinal measure of agreement/disagreement with a statement. Typically several are averaged to produce an overall measure of a person's perspective or attitude on some subject, for which the responses all relate to. I am curious how you will model that using CCO. For example, what is a measure of agreement about? Assuming here that you want to model those datums, and not the questionnaire itself. 

Initial thoughts on the scale:
Introduce a new term cco:RatingScale as a subclass of cco:ReferenceSystem. Add new term cco:LikertScale and make it and cco:PriorityScale subs of cco:RatingScale. Instances of LikertScale could be differentiated based on number of levels and values used. Perhaps the number of levels could be subclasses, e.g, 'FiveLevelLikertScale', with instances differentiated based on particular values used. The instances then are realted to IBEs using cco:uses_reference_system. Thoughts?

As to whether all these, incl. Measurement Units, are prescriptive rather than descriptive, I need to think about that some more.

**Comment by neilotte** [comment]
@mark-jensen I'm optimistic we can do justice to the averaging of responses if we can represent the individual report. e.g. there would be an Intentional Act that applies a statistical operation to a set of individual reports (each input in the intentional act) to create (has output) a new measurement that is about a disposition inhering in the aggregate of participants. 

Your initial thoughts make sense to me. Some additional ones:

I don't know if 'rating scale' implies either ordinal or ratio scales only; if it does, there might be a need to generalize so that categorical and dichotomous scales can be included without placing them under 'reference system'. Here, one might just use 'scale'. 

The taxonomy of likert scales could be done in a number of different ways (e.g. Likert scales can be distinguished by whether they are ordinal or interval; whether they can be converted to an interval scale (some can, some can't); by how many points they have (2 - 10 being common); and whether they have a mid point.) Fortunately, I don't have a present need to represent it all.   

Likert scales are typically parts of questionnaires, and the same scale can be used in different questionnaires. I suppose something worth anticipating here is the relationship between a questionnaire and a scale. Shooting from the hip, I'd use parthood and say: 'this instance of a questionnaire has part (at all times) this instance of likert scale.' Once the questionnaire is filled in with responses, it ceases to be a questionnaire, and is now a record of responses to the questionnaire. Presumably this parthood relation would also hold between the record of responses and the same likert scale. But whether parthood is appropriate here--as among other ICEs--I'm less certain of.

**Comment by alanruttenberg** [comment]
For reference, from [OBI](http://www.ontobee.org/ontology/OBI): [categorical measurement datum](http://purl.obolibrary.org/obo/OBI_0000938), [categorical value specification](http://purl.obolibrary.org/obo/OBI_0001930),  [categorical label](http://purl.obolibrary.org/obo/OBI_0000963)
